packages/tokens/FigmaToCunningham.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * This util script is used to convert Figma's Token Studio JSON color config to TS file.
+ *
+ * CLI usage example:
+ *  $ yarn run ts-node FigmaToCunningham.ts dark.json
```

Comment:
What do you think to add a command in the package.json ? Putting this command into package will explicit its existence I think.

```
figmaToCunningham: yarn run ts-node FigmaToCunningham.ts
```

Then :
```
$> yarn run figmaToCunningham dark.json
```

Reply:
Great idea!